src/main/java/redis/clients/jedis/JedisCluster.java

```diff
@@ -1089,26 +1089,6 @@ public Long execute(Jedis connection) {
   }
 
   @Override
-  public List<String> blpop(final String arg) {
```

Comment:
Sure. Doing it.

Jungtaek Lim <notifications@github.com> escreveu no dia ter, 3/03/2015 às
11:23:

> In src/main/java/redis/clients/jedis/JedisCluster.java
> <https://github.com/xetorthio/jedis/pull/902#discussion_r25690235>:
>
> > @@ -1089,26 +1089,6 @@ public Long execute(Jedis connection) {
> >    }
> >
> >    @Override
> > -  public List<String> blpop(final String arg) {
>
> Oh, that would be nice. Could you do it if you don't mind?
> Thanks in advance!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/xetorthio/jedis/pull/902/files#r25690235>.
>